### PR TITLE
Add support for reportchars=A (`-rA`)

### PR DESCRIPTION
**Pull request by blueyed**
TODO:

- [x] changelog, docs

### CODE REVIEW

**Comment by blueyed** [comment]
Added doc/changelog.

File: src/_pytest/terminal.py [hunk 1]
```diff
@@ -166,10 +166,13 @@ def getreportopt(config):
...
     if reportchars:
         for char in reportchars:
-            if char not in reportopts and char != "a":
-                reportopts += char
-            elif char == "a":
+            if char == "a":
                 reportopts = "sxXwEf"
+            elif char == "A":
+                reportopts = "sxXwEfpP"
+                break
```
Comment (nicoddemus) [inline src/_pytest/terminal.py]: Should we `break` on `if char == "a":` above?

File: changelog/5068.feature.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+The ``-r`` option learnt about ``A`` to display all reports (including passed ones) in the short test summary.
```
Comment (nicoddemus) [inline changelog/5068.feature.rst]: Git-like, I like it. 😁

File: src/_pytest/terminal.py (hunk 1 above)
Comment (blueyed) [reply src/_pytest/terminal.py]: That would not work for `aA` then.. ;)

File: changelog/5068.feature.rst (hunk 2 above)
Comment (blueyed) [reply changelog/5068.feature.rst]: Heh, noticed this myself.. :)

File: src/_pytest/terminal.py (hunk 1 above)
Comment (nicoddemus) [reply src/_pytest/terminal.py]: Ahh right, thanks! 👍

**Pull request merged**